python/helios/scene.py

```diff
@@ -97,14 +97,36 @@ def from_xml(cls, scene_part_file: AssetPath, id: int):
     @classmethod
     @validate_call
     def from_obj(cls, obj_file: AssetPath, up_axis: Literal["y", "z"] = "z"):
-        """Load the scene part from an OBJ file."""
+        """Load the scene part from an OBJ file.
+
+        For paths (potentially) containing wildcards, use 'ScenePart.from_objs()' instead!
+        """
 
         _cpp_part = _helios.read_obj_scene_part(
             str(obj_file), [str(p) for p in get_asset_directories()], up_axis
         )
 
         return cls._from_cpp(_cpp_part)
 
+    @classmethod
+    @validate_call
+    def from_objs(cls, obj_files: MultiAssetPath, up_axis: Literal["y", "z"] = "z"):
+        """Load multiple scene parts from OBJ files
+
+        Expects a single Path containing some (or none) wildcards ('*').
+        Supports '**' for matching multiple directories.
+        """
+
+        _cpp_parts = []
+        for obj in obj_files:
+            _cpp_parts.append(
+                _helios.read_obj_scene_part(
+                    str(obj), [str(p) for p in get_asset_directories()], up_axis
+                )
+            )
+
+        return [cls._from_cpp(part) for part in _cpp_parts]
```

Comment:
Couldn't we just implement this via `[ScenePart.from_obj(f) for f in obj_files]`. THat would not duplicate the underlying logic.